script.js

```diff
@@ -25,6 +25,13 @@ idApp.eventListeners = function() {
       // Call the idApp.finalDisplay function, passing in finalResult array as an argument 
       idApp.finalDisplay(finalResult);
    })
+
+   // Event Listener #3 - for when user hits the "Show Me More" button
+   $('main').on('submit', '.refreshOptions', function(event) {
+      event.preventDefault();
+      $('.resultsList').html('');
```

Comment:
we need to fix the refresh issue with the height of the element collapsing, this should probably be it's own branch